public/js/reducers/tests/ui.js

```diff
@@ -0,0 +1,69 @@
+declare var describe: (name: string, func: () => void) => void;
+declare var it: (desc: string, func: () => void) => void;
+
+const {
+  State, update } = require("../ui");
+const expect = require("expect.js");
+const C = require("../../constants");
+
+import type { SidebarsState } from "../ui";
+
+describe("ui reducer", () => {
```

Comment:
This could probably be an *action* test. The benefit would be that we can then exercise the actions and the selectors.

i.e.

```js
expect(getSidebarCollapsed("left")).to.be(false)
toggleSidebar("left");
expect(getSidebarCollapsed("left")).to.be(true)
```